viewer/src/components/Charts.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="gaia-viewer-modal">
+    <div id="player-selection">
+      <span
+        v-for="index in players"
+        :key="index"
+        @click="
+          $data.selected = index;
+          selectPlayer(index);
```

Comment:
No need to use `$data`. And better to do it in `selectPlayer`, with `this.selected = index`